cmd/do-csi-plugin/main.go

```diff
@@ -40,6 +40,7 @@ func main() {
 		defaultVolumesPageSize = flag.Uint("default-volumes-page-size", 0, "The default page size used when paging through volumes results (default: do not specify and let the DO API choose)")
 		doAPIRateLimitQPS      = flag.Float64("do-api-rate-limit", 0, "Impose QPS rate limit on DigitalOcean API usage (default: do not rate limit)")
 		version                = flag.Bool("version", false, "Print the version and exit.")
+		validateAttachment     = flag.Bool("validate-attachment", false, "Validate if the attachment is in a running state.")
```

Comment:
Similar to my other comment about hiding the "runningness" nature of a device, I'd suggest we formulate the description here as

```
Validate if the attachment has fully completed before formatting/mounting the device
```

(nit: no trailing punctuation.)

Reply:
Removed the trailing punctuation and used the above message.